specs/factorial_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative '../lib/factorial'
+
+describe "factorial" do
+  describe "basic tests" do
+    describe "test 1" do
```

Comment:
Same as the other repos, we should probably remove the `describe` blocks that are immediate parents of individual tests.

Reply:
Done.